docs/CHANGELOG.md

```diff
@@ -1,29 +1,59 @@
+ * Changes in the trace buffering logic (#1151) :
+   * Traces are now serialized as soon as possible, instead of every second. This reduces the lifetime of Span objects, which in turn should decrease the number of gen 1/2 garbage collections
+   * Whenever adding a trace would cause the buffer to overflow, the contents are immediately flushed. This should reduce the number of dropped traces for customers with a very large amount of spans
+ * Optimizations in the native profiler (#1224, #1217, #1215)
+ * Duck-typing: rename typing cast methods to better reflect the intent (#1220), and add a `DuckIgnore` attribute (#1257)
+ * Disable log rate limit when debug logging is enabled (#1239)
+ * CallTarget instrumentation:
+    * Add support for Redis (#1230)
+    * Add support for GraphQL (#1241)
+    * Add support for MongoDB (#1214)
+    * Add support for ASP.NET MVC and WebAPI (#1208)
+    * Add support for CurlHandler (#1252)
+    * Add support for Elasticsearch (#1248)
+    * Add support for RabbitMQ (#1186)
+    * Add support for WCF (#1272)
+    * Refactor HttpMessageHandler-based instrumentations (#1258) and enable them by default (#1277)
+    * Add fast-path for integrations with 7 or 8 parameters (#1261)
+    * Enable inlining by default (#1276)
+    * Change log severity (#1278)
+
+ * Various changes to CI integration (#1242, #1247, #1251, #1244)
+
+## Fixes
+ * Fix some log messages (#1240)
+ * Status was incorrectly reported for NUnit tests with no assertions (#1235)
+ * Strengthen type check in the method resolution (#1225) and ducktyping (#1291). This should fix some `BadImageFormatException` errors when loading assemblies into different load contexts 
+ * Remove sync-over-async when communicating to the agent through named pipes in AAS (#1218)
+ * Calltarget:
+    * Don't call `FindMemberRef` when the signature is empty (#1259)
+    * Remove useless instruction in the emitted IL (#1267)
+    * Properly return a faulted task when an exception is thrown in an instrumented async method (#1270)
+
+## Build / Test
+ * Update Moq to version 4.16.0 and Xunit to version 2.4.1 (#1227, #1231)
+ * Update .NET SDK version to 5.0.103 (#1237)
+ * Update log4net to 2.0.12 (#1243)
+ * Fix Xunit serialization in tests (#1236)
+ * Update the automatic logs injection sample apps (#1195)
+ * Add a Service Fabric sample app (#1190)
+ * Improve ASP.NET integration tests (#1246)
+ * Fix solution load deadlock for Rider on non-Windows OS (#1256)
+ * Fix build errors in CallTargetNativeTest (#1254)
+ * Update 3rd party license file (#1260)
+ * Enable WCF integration tests (#1273)
+ * Fix flaky tests (#1262, #1263, #1264, #1265, #1266)
+
+[Changes since 1.24.0](https://github.com/DataDog/dd-trace-dotnet/compare/v1.24.0...v1.25.0)
+
+[All commits](https://github.com/DataDog/dd-trace-dotnet/compare/v1.24.0...v1.25.0) | [Full diff](https://github.com/DataDog/dd-trace-dotnet/compare/v1.24.0..v1.25.0)
```

Comment:
The "changes since 1.24.0" link and the "All commits" are the same. I think we can probably drop one of them?

Reply:
I thought so at first, but actually there's one more dot in one of them 😄